.env.prod

```diff
@@ -0,0 +1,28 @@
+PORT=3000
+HTTPS=true
+
+REACT_APP_BLOCKNATIVE_DAPP_ID=
+REACT_APP_SUPPORT_SERVICE=
+REACT_APP_SUPPORT_URL=
+
+## These are only required if using Sentry for error logging
+REACT_APP_SENTRY_DSN_URL=
+REACT_APP_SENTRY_ENV=production
+SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="1"
+REACT_APP_ETHEREUM_NETWORK_ID="1"
+REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Mainnet"
+REACT_APP_ETHEREUM_BRIDGE_ADDRESS="0xCaF65AB2eC9B39403966991eb34B1e8B9E44C041"
+REACT_APP_ETHEREUM_ERC20_HANDLER_ADDRESS="0x8fe028Eb002bbc3ec45c5dF8acfFf67eC95B6f88"
+REACT_APP_ETHEREUM_RPC_URL=""
+REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS="0x2da719db753dfa10a62e140f436e1d67f2ddb0d6"
+REACT_APP_ETHEREUM_ERC20_NAME="CERE Network"
+REACT_APP_ETHEREUM_ERC20_SYMBOL="CERE"
+
+### Cerebellum Network
+REACT_APP_CERE_CHAIN_ID=11

```

Comment:
Why `11`?

Reply:
As we discussed in call, I kept it as 11

Reply:
We discussed that you will try this locally to check our theory but later you will research the official documentation about chain ids generation.